include/os/freebsd/spl/sys/uio.h

```diff
@@ -35,55 +35,68 @@
 #include <sys/_uio.h>
 #include <sys/debug.h>
 
-
-
 #define	uio_loffset	uio_offset
```

Comment:
You could remove this as well and adjust the accessor below accordingly.

Reply:
Good catch. I completely missed this.

Reply:
I have updated this.